modules/bibupload/lib/bibupload.py

```diff
@@ -2274,11 +2276,11 @@ def _process_document_moreinfos(more_infos, docname, version, docformat, mode):
 
 ### Update functions
 
-def update_bibrec_date(now, bibrec_id, insert_mode_p, pretend=False):
+def update_bibrec_date(now, bibrec_id, insert_mode_p, original_005=None, pretend=False):
     """Update the date of the record in bibrec table """
     if insert_mode_p:
         query = """UPDATE bibrec SET creation_date=%s, modification_date=%s WHERE id=%s"""
-        params = (now, now, bibrec_id)
+        params = (original_005, now, bibrec_id)
```

Comment:
Somehow this situation should not happen, because in `inser_mode_p` there is always an `original_005` (in the sense that if it is not there the outer call would have initialized it anyway...